librz/analysis/xrefs.c

```diff
@@ -209,8 +209,8 @@ RZ_API void rz_analysis_xrefs_list(RzAnalysis *analysis, int rad) {
 	}
 	rz_list_foreach (list, iter, xref) {
 		int t = xref->type ? xref->type : ' ';
-		switch (rad) {
-		case '*':
+		switch (mode) {
+		case RZ_OUTPUT_MODE_RIZIN:
 			analysis->cb_printf("ax%c 0x%" PFMT64x " 0x%" PFMT64x "\n", t, xref->to, xref->from);
 			break;
 		case '\0': {
```

Comment:
@XVilka Just as `q` needs to be changed to `RZ_OUTPUT_MODE_QUIET`, what should `'\0'` be changed to? A lot of the tests are failing because this warning ("0 is not enumerated in `RzOutputMode`") is being treated as an error.

Reply:
'\0' is the same as standard